### PR TITLE
Remove "</img>" - Invalid tag

### PR DESCRIPTION
**Pull request by yvisherve**

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <div class="wrapper">
       <div class="image__wrapper">
         <img class="error__illustration__bubble" alt="Gem missing from treasure chest" src="/images/cross.png">
-        <img height="530" alt="404 error" src="/images/sea_floor.svg"></img>
+        <img height="530" alt="404 error" src="/images/sea_floor.svg">
```
Comment (rietta) [inline public/404.html:13]:
```suggestion
        <img height="530" alt="404 error" src="/images/sea_floor.svg"/>
```

File: public/422.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="wrapper">
       <div class="image__wrapper">
-        <img height="530" alt="422 error" src="/images/sea_level.svg"></img>
+        <img height="530" alt="422 error" src="/images/sea_level.svg">
```
Comment (rietta) [inline public/422.html:12]:
```suggestion
        <img height="530" alt="422 error" src="/images/sea_level.svg"/>
```

File: public/500.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="wrapper">
       <div class="image__wrapper">
-        <img height="530" alt="500 error" src="/images/sea_level.svg"></img>
+        <img height="530" alt="500 error" src="/images/sea_level.svg">
```
Comment (rietta) [inline public/500.html:12]:
```suggestion
        <img height="530" alt="500 error" src="/images/sea_level.svg"/>
```

File: public/502.html [hunk 4]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="wrapper">
       <div class="image__wrapper">
-        <img height="530" alt="502 error" src="/images/sea_level.svg"></img>
+        <img height="530" alt="502 error" src="/images/sea_level.svg">
```
Comment (rietta) [inline public/502.html:12]:
```suggestion
        <img height="530" alt="502 error" src="/images/sea_level.svg"/> 
```

File: public/503.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="wrapper">
       <div class="image__wrapper">
-        <img height="530" alt="503 error" src="/images/sea_level.svg"></img>
+        <img height="530" alt="503 error" src="/images/sea_level.svg">
```
Comment (rietta) [inline public/503.html:12]:
```suggestion
        <img height="530" alt="503 error" src="/images/sea_level.svg"/>
```

File: public/maintenance/index.html [hunk 6]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="wrapper">
       <div class="image__wrapper">
-        <img height="530" alt="under maintenance" src="/images/gem_wrench.svg"></img>
+        <img height="530" alt="under maintenance" src="/images/gem_wrench.svg">
```
Comment (rietta) [inline public/maintenance/index.html:12]:
```suggestion
        <img height="530" alt="under maintenance" src="/images/gem_wrench.svg"/>
```

**Comment by yvisherve** [comment]
@rietta I think the standard here is HTML5.

1/ The doctype used in all of these files is the doctype of an HTML5 document.

`<!DOCTYPE html>`

2/ **&lt;img>** is used in the file **/public/404.html**.

`Gem missing from treasure chest`

This is HTML5 valid but not XHTML valid.

3/ **&lt;img>** tends to be more widely used than **&lt;img/>**. 

In the [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img) **&lt;img>** is used instead of **&lt;img/>**.

You could also inspects which tag is used on [http://google.com](http://google.com) or most other websites.

Anyway both are HTML5 valid, but I prefer **&lt;img>**.

**Comment by simi** [comment]
@rietta :pray: XHTML days are finally over.

**Pull request merged**